lib/data/sql/V3__create_types.sql

```diff
@@ -834,6 +834,45 @@ FROM nat_boden
 LEFT JOIN nat_naistyp ON nat_boden.naistyp_sort = nat_naistyp.naistyp_sort
 WHERE trim(nat_naistyp.naistyp_c) = foresttype_meta.code;
 
+-- ALTER TABLE lu_standorttypen
```

Comment:
Please remove disabled code.

Reply:
sry, I sometimes still forget to review my own code =/